opm/simulators/utils/ParallelRestart.cpp

```diff
@@ -339,7 +539,7 @@ std::size_t packSize(const RestartValue& data, Opm::Parallel::MPIComm comm)
 
 std::size_t packSize(const Opm::time_point&, Opm::Parallel::MPIComm comm)
 {
-    std::time_t tp;
+    std::time_t tp = 0;
```

Comment:
Extremely tiny nit:  Although `std::time_t` is guaranteed to be an arithmetic type, it is not **necessarily** guaranteed to support initialization from the integer literal `0`.  If we're being **very** picky, we should either write this as
```C++
std::time_t tp{};
```
or as
```C++
std::time_t tp = std::time(nullptr);
```
On Posix systems, of course, `std::time_t` is a signed integer type representing the number of seconds since midnight GMT 1 January 1970 so `= 0` is a perfectly good initializer on those systems.